Microsoft.HealthVault/Configuration/HealthVaultConfigurationDefaults.cs

```diff
@@ -36,9 +36,9 @@ public class HealthVaultConfigurationDefaults
         public static readonly Uri RestUrl = new Uri("https://data.microsofthealth.net");
 
         /// <summary>
-        /// Default sleep duration in seconds.
+        /// Default sleep duration.
         /// </summary>
-        public static readonly TimeSpan RetryOnInternal500SleepDuration = TimeSpan.FromMinutes(1);
+        public static readonly TimeSpan RetryOnInternal500SleepDuration = TimeSpan.FromSeconds(1);
```

Comment:
Also, confirmed that the classic .NET SDK matches those docs and uses one second, not one minute.